modules/ximgproc/samples/edgepreserving_filter_demo.cpp

```diff
@@ -0,0 +1,47 @@
+#include <iostream>
+#include <string>
+#include <opencv2/imgcodecs.hpp>
+#include <opencv2/highgui.hpp>
+#include <opencv2/ximgproc.hpp>
+#include <opencv2/opencv.hpp>
+#include <opencv2/epf.hpp>
```

Comment:
this does not exist, right ? ( `#include <opencv2/ximgproc.hpp>` should be enouh, here)